iOS/DuckDuckGo/Subscription/UserScripts/SubscriptionPagesUseSubscriptionFeature.swift

```diff
@@ -68,14 +69,12 @@ enum UseSubscriptionError: Error {
          missingEntitlements,
          failedToGetSubscriptionOptions,
          failedToSetSubscription,
-         failedToRestoreFromEmail,
-         failedToRestoreFromEmailSubscriptionInactive,
-         failedToRestorePastPurchase,
-         subscriptionNotFound,
-         subscriptionExpired,
-         hasActiveSubscription,
          cancelledByUser,
          accountCreationFailed,
+         activeSubscriptionAlreadyPresent,
+         restoreFailedDueToNoSubscription,
+         restoreFailedDueToExpiredSubscription,
+         otherRestoreError,
          generalError
```

Comment:
Removed unused, and renamed and reordered rest of the errors to align them with macOS implementation.